linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_data_retention_flush/rule.yml

```diff
@@ -33,6 +33,7 @@ references:
     cobit5: APO10.01,APO10.03,APO10.04,APO10.05,APO11.04,BAI03.05,DSS01.03,DSS03.05,DSS05.02,DSS05.04,DSS05.05,DSS05.07,MEA01.01,MEA01.02,MEA01.03,MEA01.04,MEA01.05,MEA02.01
     iso27001-2013: A.12.4.1,A.12.4.2,A.12.4.3,A.12.4.4,A.12.7.1,A.14.2.7,A.15.2.1,A.15.2.2
     cis-csc: 1,12,13,14,15,16,2,3,5,6,7,8,9
+    srg: SRG-OS-000051-GPOS-00024
```

Comment:
Nack. The control is for "The operating system must provide the capability to centrally review and analyze audit records from multiple components within the system."

Even if flush and freq are configured, RHEL would always have the capability to review and analyze udit records. Better rule to map these to could be "is audit utils installed?"

Reply:
I used this SRG since the rule has "If the value is not set  audit records could fail to be is fully synchronized with the log files on the disk."  

If the data is not synchronized analysis could be more problematic.  

Let me know what you think.  I can change it if needed.


Is there a better SRG to map the rule to?

Reply:
I can't see the actual check, but if this is setting the flush to INCREMENTAL_ASYNC, then you want this. It has the highest performance with a predictable safety margin in the event of loss of logging. In the past, people put sync and this kills their system.

Reply:
Remapped to generic SRG-OS-000480-GPOS-00227